### ISSUE TITLE
Scale and rotate selected objects

### ISSUE THREAD

**Issue by boniatillo-com** [boniatillo-com/PhaserEditor#17]
@ponderboy

Selecting more then one sprite and being able to scale and rotate them would be very helpful as well.

**Comment by codingisnothingtodo** [comment]
And also fixing proportion pressing shift key when resizing will be better I think.

**Comment by PhaserEditor2D** [comment]
Now with shift, it scales proportionally. Yet I have to implement multiple selections.

**Comment by codingisnothingtodo** [comment]
Well done Arian. In fact I use group for resizing multi objects.

2018-02-14 19:39 GMT+03:00 Phaser Editor <notifications@github.com>:

**Comment by PhaserEditor2D** [comment]
Yes, but it is true that sometimes you would like to scale some of them at the same time, even inside the same group.
